test-network/scripts/deployCC.sh

```diff
@@ -1,255 +1,307 @@
+	# first see which chaincode we have. This will be based on the
+	# short name of the known chaincode sample
+	if [ "$CC_NAME" = "basic" ]; then
+		echo $'\e[0;32m'asset-transfer-basic$'\e[0m' chaincode
+		CC_SRC_PATH="../asset-transfer-basic"
+	elif [ "$CC_NAME" = "secure" ]; then
+		echo $'\e[0;32m'asset-transfer-secured-agreeement$'\e[0m' chaincode
+		CC_SRC_PATH="../asset-transfer-secured-agreement"
+	elif [ "$CC_NAME" = "ledger" ]; then
+		echo $'\e[0;32m'asset-transfer-secured-agreeement$'\e[0m' chaincode
+		CC_SRC_PATH="../asset-transfer-ledger-queries"
+	elif [ "$CC_NAME" = "private" ]; then
+		echo $'\e[0;32m'asset-transfer-private-data$'\e[0m' chaincode
+		CC_SRC_PATH="../asset-transfer-private-data"
+	else
+		echo The chaincode name ${CC_NAME} is not supported by this script
+		echo Supported chaincode names are: basic, secure, and private
+		exit 1
+	fi
+
+	# now see what language it is written in
+	if [ "$CC_SRC_LANGUAGE" = "go" -o "$CC_SRC_LANGUAGE" = "golang" ]; then
+		CC_SRC_PATH="$CC_SRC_PATH/chaincode-go/"
+	elif [ "$CC_SRC_LANGUAGE" = "java" ]; then
+		CC_SRC_PATH="$CC_SRC_PATH/chaincode-java/"
+	elif [ "$CC_SRC_LANGUAGE" = "javascript" ]; then
+		CC_SRC_PATH="$CC_SRC_PATH/chaincode-javascript/"
+	elif [ "$CC_SRC_LANGUAGE" = "typescript" ]; then
+		CC_SRC_PATH="$CC_SRC_PATH/chaincode-typescript/"
+	fi
+fi
+
+# do some language specific preparation to the chaincode before packaging
+if [ "$CC_SRC_LANGUAGE" = "go" -o "$CC_SRC_LANGUAGE" = "golang" ]; then
+		CC_RUNTIME_LANGUAGE=golang
 
-	echo Vendoring Go dependencies ...
-	pushd ../chaincode/fabcar/go
+	echo Vendoring Go dependencies at $CC_SRC_PATH
+	pushd $CC_SRC_PATH
 	GO111MODULE=on go mod vendor
 	popd
 	echo Finished vendoring Go dependencies
 
-elif [ "$CC_SRC_LANGUAGE" = "javascript" ]; then
-	CC_RUNTIME_LANGUAGE=node # chaincode runtime language is node.js
-	CC_SRC_PATH="../chaincode/fabcar/javascript/"
-
 elif [ "$CC_SRC_LANGUAGE" = "java" ]; then
 	CC_RUNTIME_LANGUAGE=java
-	CC_SRC_PATH="../chaincode/fabcar/java/build/install/fabcar"
 
 	echo Compiling Java code ...
-	pushd ../chaincode/fabcar/java
+	pushd $CC_SRC_PATH
 	./gradlew installDist
 	popd
 	echo Finished compiling Java code
+	CC_SRC_PATH=$CC_SRC_PATH/build/install/$CC_NAME
+
+elif [ "$CC_SRC_LANGUAGE" = "javascript" ]; then
+	CC_RUNTIME_LANGUAGE=node
 
 elif [ "$CC_SRC_LANGUAGE" = "typescript" ]; then
-	CC_RUNTIME_LANGUAGE=node # chaincode runtime language is node.js
-	CC_SRC_PATH="../chaincode/fabcar/typescript/"
+	CC_RUNTIME_LANGUAGE=node
 
 	echo Compiling TypeScript code into JavaScript ...
-	pushd ../chaincode/fabcar/typescript
+	pushd $CC_SRC_PATH
 	npm install
 	npm run build
 	popd
 	echo Finished compiling TypeScript code into JavaScript
 
 else
 	echo The chaincode language ${CC_SRC_LANGUAGE} is not supported by this script
-	echo Supported chaincode languages are: go, java, javascript, and typescript
+	echo Supported chaincode languages are: go, golang, java, javascript, and typescript
```

Comment:
Why add 'golang'? why not just keep as 'go' everywhere?

Reply:
removed